packages/nouns-contracts/contracts/NounsArt.sol

```diff
@@ -0,0 +1,449 @@
+// SPDX-License-Identifier: GPL-3.0
+
+/// @title The Nouns art storage contract
+
+/*********************************
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░██░░░████░░██░░░████░░░ *
+ * ░░██████░░░████████░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ *********************************/
+
+pragma solidity ^0.8.6;
+
+import { INounsArt } from './interfaces/INounsArt.sol';
+import { Inflate } from './libs/Inflate.sol';
+import { SSTORE2 } from './libs/SSTORE2.sol';
+
+contract NounsArt is INounsArt {
+    /// @notice Current Nouns Descriptor address
+    address public override descriptor;
+
+    /// @notice Pending Nouns Descriptor address
+    address public override pendingDescriptor;
```

Comment:
not a big deal but a consideration is using "migration" related variable/methods for readability